iosApp/SurveyUITests/Sources/Screens/ScreenProtocol.swift

```diff
@@ -72,3 +93,23 @@ extension ScreenProtocol {
         button.tap()
     }
 }
+
+// MARK: Wait
+
+extension ScreenProtocol {
+
+    func wait(
+        timeout: TimeInterval = 10,
```

Comment:
Can we use `.default` here? I noticed we created the `default` constant for `TimeInterval` as `10` already, right? 🤔

Reply:
Updated 👍